pkg/etcdenvvar/envvarcontroller.go

```diff
@@ -137,6 +151,10 @@ func (c *EnvVarController) checkEnvVars() error {
 		return err
 	}
 
+	for _, status := range operatorStatus.NodeStatuses {
+		c.revisionGauge.WithLabelValues(status.NodeName).Set(float64(status.CurrentRevision))
```

Comment:
[Doesn't seem to work][1].  I think you're missing the piping-it-into-channel, like [this][2].  Although I'm a bit fuzzy on how the existing etcd-operator metrics are wired up.

[1]: https://github.com/openshift/cluster-etcd-operator/pull/852#discussion_r894935496
[2]: https://github.com/openshift/cluster-etcd-operator/blob/c9977ae3bd788a9e9a595001c5bc24995f9d0175/pkg/etcdcli/health.go#L278-L289

Reply:
Hrm, yeah, it looks like [that][1] is one way to do things.  And see [here][1] for an example using `Collect`.  Might need to ask the monitoring folks to help figure out why your current code doesn't seem to be working.

[1]: https://pkg.go.dev/github.com/prometheus/client_golang/prometheus#hdr-A_Basic_Example
[2]: https://pkg.go.dev/github.com/prometheus/client_golang/prometheus#Collector

Reply:
Hmm, where are you setting up your metrics listener?  Grepping around in the operator code, I'm not finding it...